xep-0372.xml

```diff
@@ -25,6 +25,12 @@
   <supersededby/>
   <shortname>Refs</shortname>
   &ksmithisode;
+  <revision>
+    <version>0.3</version>
+    <date>2019-09-04</date>
+    <initials>(mb)</initials>
+    <remark>Make Pubsub URIs conform to RFC5222 in sections 3.3 and 3.4</remark>
```

Comment:
This should be RFC 6122? You want to use the `&rfc6122;` XML entity reference for this. And maybe you also want to reference `&xep0147;`.

Reply:
Gah. I meant 5122 of course.

Reply:
You cannot use those entities in ``<remark/>``:

```
xep-0372.xml:32: element remark: validity error : Element span is not declared in remark list of possible children

  <remark>Make Pubsub URIs in sections 3.3 and 3.4 conform to &xep0147;.</remark

                                                                               ^

xep-0372.xml:32: element remark: validity error : Element note is not declared in remark list of possible children

  <remark>Make Pubsub URIs in sections 3.3 and 3.4 conform to &xep0147;.</remark

                                                                               ^

Makefile:110: recipe for target '/var/www/html/extensions//xep-0372.html' failed
```